src/query/src/sql/show_create_table.rs

```diff
@@ -126,6 +135,10 @@ fn create_table_constraints(
     constraints
 }
 
+fn is_internal_column(name: &str) -> bool {
```

Comment:
It's better to move this function into `store_api::metric_engine_consts`, it can be used in some places.